GUI/src/main/java/put/io/black/java/gui/ScenarioGUI.java

```diff
@@ -131,87 +192,79 @@ public static void main(String args[]) {
-            HttpURLConnection conn = (HttpURLConnection) url.openConnection();
-            conn.setRequestMethod("POST");
-            conn.setRequestProperty("Accept", "application/json");
-            conn.setRequestProperty("Content-Type", "application/json");
-
-            if (conn.getResponseCode() != 200) {
-                throw new RuntimeException("Failed : HTTP error code : "
-                        + conn.getResponseCode());
+            URL url = new URL(serverIP + endpoint);
+            HttpURLConnection connection = (HttpURLConnection) url.openConnection();
+
+            connection.setDoInput(true);
+            connection.setDoOutput(true);
+            connection.setRequestMethod("POST");
+            connection.setRequestProperty("Accept", "application/json");
+            connection.setRequestProperty("Content-Type", "application/json; charset=UTF-8");
+            OutputStreamWriter writer = new OutputStreamWriter(connection.getOutputStream(), "UTF-8");
+            writer.write(source);
+            writer.close();
+            BufferedReader br = new BufferedReader(new InputStreamReader(connection.getInputStream()));
+            StringBuffer jsonString = new StringBuffer();
+            String line;
+            while ((line = br.readLine()) != null) {
+                jsonString.append(line);
             }
-            logger.warning("Connected!");
-            BufferedReader br = new BufferedReader(new InputStreamReader(
-                    (conn.getInputStream())));
-
-            String output;
-            logger.warning("Output from Server ....");
-            while ((output = br.readLine()) != null) {
-                logger.warning(output);
-                outputF += output;
+            br.close();
+            connection.disconnect();
+
+            JsonObject jsonObject = new JsonParser().parse(jsonString.toString()).getAsJsonObject();
+            JsonElement status = jsonObject.get("status");
+            if (status != null) {
+                if (status.getAsString().trim().equals("success")) {
+                    return jsonObject.get("result").getAsString();
+                } else {
+                    return jsonObject.get("message").getAsString();
+                }
+            } else {
+                return "No return from server";
             }
-            conn.disconnect();
-            logger.warning("Disconnected!");
-        } catch (MalformedURLException e1) {
-            e1.printStackTrace();
-            logger.warning("MalformedURLException - see Stack");
-        } catch (IOException e1) {
-            e1.printStackTrace();
-            logger.warning("IOException - see Stack");
+        } catch (Exception e) {
+            throw new RuntimeException(e.getMessage());
         }
-        return outputF;
     }
 
     {
-    // GUI initializer generated by IntelliJ IDEA GUI Designer
-    // DO NOT EDIT OR ADD ANY CODE HERE!
         $$$setupUI$$$();
```

Comment:
Maybe add comment here, that it is from IntelliJ, and not delete this?

Reply:
I need delete this when I prepare javadocs because plugin failed (normally added with `>>>>IMPORTANT<<<<<`).